docs/fundamentals/apicompat/diagnostic-ids.md

```diff
@@ -61,7 +61,7 @@ This reference article lists all the error codes generated by package validation
 | CP0009 | A type that was unsealed on one side was annotated as `sealed` on the other compared side. | Remove the `sealed` annotation from the type. |
 | CP0010 | The underlying type of an enum changed from one side to the other. | Change the underlying type back to what it was previously. |
 | CP0011 | The value of a member in an enum changed from one side to the other. | Change the value of the member back to what it was previously. |
-| CP0012 | The `virtual` keyword was removed from a member that was previously virtual. | Add the `virtual` keyword back to the member. |
+| CP0012 | Either the `virtual` or `abstract` keyword was removed from a member that was previously virtual or abstract. | Add the `virtual` or `abstract` keyword back to the member. |
```

Comment:
We could mention that it's safe to move from `abstract` to `virtual`.

Reply:
@ericstj Do you think we should also mention it in the exception message, or only here?

Reply:
@gewarren what do you think?:

```suggestion
| CP0012 | Either the `virtual` or `abstract` keyword was removed from a member that was previously virtual or abstract. | Add the `virtual` or `abstract` keyword back to the member. Note that replacing `abstract` with `virtual` is allowed. |
```

Reply:
I think that's fine, or perhaps this is even clearer:

```suggestion
| CP0012 | Either the `virtual` or `abstract` keyword was removed from a member that was previously virtual or abstract. | If the member was previously virtual, add the `virtual` keyword back. If the member was previously abstract, add either the `virtual` or `abstract` keyword to the member. |
```

Reply:
Yeah that's better. Applied.